packages/components/src/Slider/Slider.component.js

```diff
@@ -219,17 +224,19 @@ class Slider extends React.Component {
 		} = this.props;
 		const noValue = value === null || value === undefined;
 		return (
-			<span className={classnames(theme['tc-slider'], 'tc-slider')}>
-				<RcSlider
-					id={id}
-					value={value}
-					min={min}
-					max={max}
-					handle={noValue ? undefined : this.state.handle}
-					className={classnames(theme['tc-slider-rc-slider'], 'tc-slider-rc-slider')}
-					onChange={onChange}
-					{...rest}
-				/>
+			<div>
```

Comment:
why double div nesting ? one only

Reply:
Need the slider padding applied only to the slider not the icons.